lib/PHPUnitTagProvider.coffee

```diff
@@ -0,0 +1,69 @@
+            {name: '@backupStaticAttributes',         documentationName: 'backupStaticAttributes',         snippet: '@backupStaticAttributes ${1:disabled}',        description: 'Back up all static property values in all declared classes before each test and restore them afterwards'}
+            {name: '@before',                         documentationName: 'before',                         snippet: '@before',                                      description: 'The method should be called before each test method in a test case class'}
+            {name: '@beforeClass',                    documentationName: 'beforeClass',                    snippet: '@beforeClass',                                 description: 'The static method should be called before any test methods in a test class are run to set up shared fixtures'}
+            {name: '@codeCoverageIgnore',             documentationName: 'codeCoverageIgnore',             snippet: '@codeCoverageIgnore',                          description: 'Exclude lines of code from the coverage analysis'}
+            {name: '@codeCoverageIgnoreEnd',          documentationName: 'codeCoverageIgnoreEnd',          snippet: '@codeCoverageIgnoreEnd',                       description: 'Exclude lines of code from the coverage analysis'}
+            {name: '@codeCoverageIgnoreStart',        documentationName: 'codeCoverageIgnoreStart',        snippet: '@codeCoverageIgnoreStart',                     description: 'Exclude lines of code from the coverage analysis'}
+            {name: '@covers',                         documentationName: 'covers',                         snippet: '@covers ${1:method}',                          description: 'Specify which method(s) the test method wants to test'}
+            {name: '@coversDefaultClass',             documentationName: 'coversDefaultClass',             snippet: '@coversDefaultClass ${1:class}',               description: 'Specify a default namespace or class name for @covers annotation'}
+            {name: '@coversNothing',                  documentationName: 'coversNothing',                  snippet: '@coversNothing',                               description: 'Specify no code coverage information will be recorded for the annotated test case'}
+            {name: '@dataProvider',                   documentationName: 'dataProvider',                   snippet: '@dataProvider ${1:provider}',                  description: 'Specify the data provider method'}
+            {name: '@depends',                        documentationName: 'depends',                        snippet: '@depends ${1:test}',                           description: 'Specify the test method this test depends'}
+            {name: '@expectedException',              documentationName: 'expectedException',              snippet: '@expectedException ${1:exception}',            description: 'Sepcify the exception that must be thrown inside the test method'}
```

Comment:
Minor nitpick: did you mean `Specify` instead of `Sepcify` on this line and the following ones? 😄